packages/terra-button/tests/wdio/button-spec.js

```diff
@@ -1,230 +1,298 @@
 /* global browser, Terra, before */
-const ignoreContrast = {
-  'color-contrast': { enabled: false },
+const shouldTheme = (customProperties, selector) => {
```

Comment:
When we create the utils package, we could stick this function there for reuse.

Reply:
This function is actually [already available as a reusable function](https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/TerraService.js#L69) in the toolkit. We need to enhance this function however to allow one to pass in the selector. Currently it defaults to [data-reactroot] and these tests specify certain butttons.